.gitattributes

```
@@ -3,6 +3,7 @@
 /demo               export-ignore
 /demo-php8          export-ignore
 /.gitattributes     export-ignore
+/.github            export-ignore
```

`vendor-bin` も追加した方が良いですね

```suggestion
/.github            export-ignore
/vendor-bin         export-ignore
```